lms/templates/problem.html

```diff
@@ -90,6 +90,7 @@ <h3 class="hd hd-3 problem-header" id="${ short_id }-problem-title" aria-describ
         % if attempts_allowed and (not submit_disabled_cta or attempts_used == 0):
           ${ngettext("You have used {num_used} of {num_total} attempt", "You have used {num_used} of {num_total} attempts", attempts_allowed).format(num_used=attempts_used, num_total=attempts_allowed)}
         % endif
+        <div>${Text(_("Grading method: {grading_method}")).format(grading_method=grading_method)}</div>
```

Comment:
When using highest score as the grading method, but the latest attempt is incorrect, this looks like this:
![image](https://github.com/openedx/edx-platform/assets/64440265/f71ae207-f38f-466e-8e2f-72f748ae2fdf)
Which in my opinion, it's a bit confusing. Was this considered during the product review?

Reply:
The image may seem a bit confusing until you read the label that says that the grading method being used is "Highest Score". I agree this can go forward as is for now.